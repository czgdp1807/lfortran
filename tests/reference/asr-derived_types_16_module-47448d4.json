{
    "basename": "asr-derived_types_16_module-47448d4",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/derived_types_16_module.f90",
    "infile_hash": "530d91f6971b7a725d7bc0e0de0f21be8593526c36d46574bd064fd4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-derived_types_16_module-47448d4.stdout",
    "stdout_hash": "3cdb2c97bf574983a811145bbe9e2f234ddd7ac765daeb04640055f9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}