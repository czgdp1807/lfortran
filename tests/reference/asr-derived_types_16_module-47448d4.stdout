(TranslationUnit
    (SymbolTable
        1
        {
            fpm_command_line:
                (Module
                    (SymbolTable
                        2
                        {
                            fpm_run_settings:
                                (StructType
                                    (SymbolTable
                                        3
                                        {
                                            bool:
                                                (Variable
                                                    3
                                                    bool
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Default
                                                    (Logical 4 [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                ),
                                            float:
                                                (Variable
                                                    3
                                                    float
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Default
                                                    (Real 4 [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                ),
                                            int:
                                                (Variable
                                                    3
                                                    int
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Default
                                                    (Integer 4 [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    fpm_run_settings
                                    []
                                    [int
                                    float
                                    bool]
                                    Source
                                    Public
                                    .false.
                                    ()
                                    ()
                                ),
                            get_command_line_settings:
                                (Function
                                    (SymbolTable
                                        4
                                        {
                                            cmd_settings:
                                                (Variable
                                                    4
                                                    cmd_settings
                                                    []
                                                    Out
                                                    ()
                                                    ()
                                                    Allocatable
                                                    (Class
                                                        2 fpm_run_settings
                                                        []
                                                    )
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    get_command_line_settings
                                    []
                                    [(Var 4 cmd_settings)]
                                    [(=
                                        (Var 4 cmd_settings)
                                        (StructTypeConstructor
                                            2 fpm_run_settings
                                            [(IntegerConstant 0 (Integer 4 []))
                                            (RealConstant
                                                1.000000
                                                (Real 4 [])
                                            )
                                            (LogicalConstant
                                                .true.
                                                (Logical 4 [])
                                            )]
                                            (Struct
                                                2 fpm_run_settings
                                                []
                                            )
                                            ()
                                        )
                                        ()
                                    )]
                                    ()
                                    Source
                                    Public
                                    Implementation
                                    ()
                                    .false.
                                    .false.
                                    .false.
                                    .false.
                                    .false.
                                    []
                                    []
                                    .false.
                                )
                        })
                    fpm_command_line
                    []
                    .false.
                    .false.
                )
        })
    []
)
